src/electionguard/key_ceremony.py

```diff
@@ -19,21 +18,53 @@
     elgamal_combine_public_keys,
     elgamal_keypair_random,
 )
-from .group import hex_to_q, rand_q, ElementModP, ElementModQ
+from .group import hex_to_q, ElementModP, ElementModQ
 from .hash import hash_elems
 from .rsa import rsa_keypair, rsa_decrypt, rsa_encrypt
-from .schnorr import SchnorrProof, make_schnorr_proof
-from .serializable import Serializable
+from .schnorr import SchnorrProof
 from .types import GUARDIAN_ID
 from .utils import get_optional
 
+ELECTION_PUBLIC_KEY = ElementModP
+ELECTION_JOINT_PUBLIC_KEY = ElementModP
+VERIFIER_ID = str
+
+
+class ElectionPublicKey(NamedTuple):
+    """A tuple of election public key and owner information"""
+
+    owner_id: GUARDIAN_ID
+    sequence_order: int
+    key: ELECTION_PUBLIC_KEY
+    coefficient_commitments: List[PUBLIC_COMMITMENT]
```

Comment:
lets pop some comments on these props, specifically to call out that the `key` is a convenience accessor for `coefficient_commitments[0]` so its clear what is happening and also maybe state that the `coefficient_proofs` are also index based and map to the commitments?